src/Text/Pretty/Simple.hs

```diff
@@ -616,12 +619,14 @@ pStringOpt outputOptions =
 --
 -- >>> pPrint $ B ( B ( B A ) )
 -- B
---     ( B ( B A ) )
```

Comment:
Again, for this and the next few, this was easier to implement.

Personally I prefer the new output here.

Reply:
(Again, I like `pretty-simple` for its compact output ;) )

Reply:
Ok, I can fix that easily enough by adding a special case for singletons (in the `Expr` type, `( A )` is treated as a unary tuple.

Reply:
Done